network/src/testutils/test_node.rs

```diff
@@ -309,6 +309,14 @@ pub trait TestNode: ApplicationNode + Sync {
             .expect("Expecting a message")
     }
 
+    /// Waits for a period of time for now message
```

Comment:
nit:
- `now` -> `no`?
- maybe: `/// Waits for a period of time and ensures no message is received`?